hub/powertoys/run.md

```diff
@@ -160,21 +165,21 @@ The PowerToys Run calculator plugin supports the following operations:
 | Division |  a / b | |
 | Modulo/Remainder | a % b | |
 | Exponentiation | a ^ b | |
-| Ceil | ceil( x.y ) |  Rounds a number up to the next larger integer. |
-| Floor | floor( x.y ) | Rounds a number down to the next smaller integer. |
+| Ceiling function | ceil( x.y ) |  Rounds a number up to the next larger integer. |
+| Floor function | floor( x.y ) | Rounds a number down to the next smaller integer. |
+| Rounding | round( x.abcd ) | _Example: `round(8.7867)`_ |
```

Comment:
Not "rounds to nearest integer" ?

Reply:
isn't this clear as you don't define how it should round?